client/me/security-2fa-setup-backup-codes/index.jsx

```diff
@@ -67,13 +41,14 @@ class Security2faSetupBackupCodes extends Component {
 	}
 
 	renderList() {
-		if ( this.state.lastError ) {
-			return null;
+		const { backupCodes } = this.props;
+		if ( ! backupCodes || backupCodes.length === 0 ) {
```

Comment:
Suggestion for brevity:

```suggestion
		if ( ! backupCodes || ! backupCodes.length ) {
```